cmd/main.go

```diff
@@ -1,11 +1,48 @@
 package main
 
 import (
+	"flag"
 	"log"
 	"time"
+
+	"github.com/lvl484/positioning-filter/config"
+	"github.com/lvl484/positioning-filter/storage"
 )
 
 func main() {
+	configPath := flag.String("cp", "../config", "Path to config file")
+	configName := flag.String("cn", "viper.config", "Name of config file")
+
+	flag.Parse()
+
+	viper, err := config.NewConfig(*configName, *configPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	consulConfig := viper.NewConsulConfig()
+	agentConfig := consulConfig.AgentConfig()
+	consulClient, err := consulConfig.NewClient()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err = consulConfig.ServiceRegister(consulClient, agentConfig); err != nil {
+		log.Fatal(err)
+	}
+
+	defer consulClient.Agent().ServiceDeregister(consulConfig.ServiceName)
+
+	postgresConfig := viper.NewDBConfig()
+	db, err := storage.Connect(postgresConfig)
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Defer call with ServiceDeregister will not work with this fatal.

Reply:
defer will not work with any fatal. But do we need to start app without db? Or we should call graceful shutdown in case of error?

Reply:
Exactly, we must return resources that we consumed from outer world.

Reply:
Even when we don't have specific graceful shutdown - we should keep application friendly to system.

Reply:
Well, this is another task that will be implemented later by another member of team

Reply:
Please, correct the behavior in this PR.

Reply:
Ok, for now i will change this to log.Println

Reply:
fixed